sites/canonical-com.yaml

```diff
@@ -0,0 +1,24 @@
+domain: canonical.com
```

Comment:
Sorry, I should have caught this much earlier. Could we please name these files by the domain name - as in with dots? Like `sites/canonical.com.yaml`?

'cos now they're not a file for a specific Kubernetes object (where the naming has hyphens) but instead a file for the whole config for a site.

Reply:
Done!